README.md

```diff
@@ -143,3 +144,25 @@ It's also possible to run the script without using docker. To this end, issue th
     export ROS_WS=/tmp/ros_ws        # define a new ROS workspace location
     mkdir $ROS_WS                    # and create it
     .moveit_ci/travis.sh
+
+## Enabling codecov.io reporting
+
+For codecov to work you need to build and link your C++ code with specific parameters.  To enable this we use the ROS package [code_coverage](https://github.com/mikeferguson/code_coverage).  To to use the `code-coverage` test in your repo make these two changes:
+
+1. Add `<test_depend>code_coverage</test_depend>` to your package.xml
+1. Add this to your `CMakeLists.txt`:
+
+```cmake
+# to run: catkin_make -DENABLE_COVERAGE_TESTING=ON package_name_coverage
+if(CATKIN_ENABLE_TESTING AND ENABLE_COVERAGE_TESTING)
+  find_package(code_coverage REQUIRED)   # catkin package ros-*-code-coverage
+  include(CodeCoverage)
+  APPEND_COVERAGE_COMPILER_FLAGS()
+  set(COVERAGE_EXCLUDES "*/test/*")
+  add_code_coverage(NAME ${PROJECT_NAME}_coverage)
```

Comment:
Is this really needed? You didn't use this command below for the unit test.

Reply:
Yes, I'll add it to the unit test.